bokehjs/src/coffee/models/transforms/interpolator.coffee

```diff
@@ -0,0 +1,85 @@
+_ = require "underscore"
+Transform = require "./transform"
+p = require "../../core/properties"
+
+class Interpolator extends Transform.Model
+
+  initialize: (attrs, options) ->
+    super(attrs, options)
+    @_x_sorted = []
+    @_y_sorted = []
+    @_sorted_dirty = true
+
+    @bind 'change', () ->
+      console.log 'Changed the configuration'
+      @_sorted_dirty = true
+
+  props: ->
+    return _.extend {}, super(), {
+      x: [ p.Any, '']
+      y: [ p.Any, '']
+      data: [ p.Any, null]
+    }
```

Comment:
FYI in master there is now a cleaner/simpler way to do this, see e.g.:

https://github.com/bokeh/bokeh/blob/master/bokehjs/src/coffee/models/ranges/data_range1d.coffee#L11-L20

Also I think the defaults for `x` and `y` should be `null` and, simply omitting a default will make `null` the default
Lastly (picky!) every other place in the lib we do line the prop defs up for readibility.